felix/calc/policy_resolver_test.go

```diff
@@ -0,0 +1,116 @@
+package calc_test
```

Comment:
I think public/private should be decided based on the API we want the object to have.  In this case, I don't think we want those fields to be public since they're not intended to be used outside the object.  Making things public to allow your tests to be in another package is tail wagging the dog.  The reason that Go allows tests to be in a separate `_test` package is to allow your public API to be tested as if you were a user of the package (and some technical point about import cycles).

Is there any other reason you want these to be in the `_test` package?

Reply:
No other reason no. Definitely agree with what you've said so I've gone ahead and simply reverted the symbol export and moved my UTs to the calc package.